modules/transactions/src/main/java/org/apache/ignite/internal/tx/InternalTransaction.java

```diff
@@ -52,14 +53,29 @@ public interface InternalTransaction extends Transaction {
      */
     TxState state();
 
+    /**
+     * Assign a replication group that will store the transaction state.
```

Comment:
Enlists (Assigns _up to you_) an id of a replication group that will store the transaction state.

Reply:
Rewrite the comment as: Assigns a replication group id to store the transaction state.

Reply:
Well, it sound like, that is replication group **id** that stores transaction state.
I mean that you assigns replication group id, that identifies a replication group that stores a transaction state.